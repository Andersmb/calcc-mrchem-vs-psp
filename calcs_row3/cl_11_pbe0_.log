Starting job 5688108 on c2-[11,14-15],c5-[29,36,60] at Thu May 19 19:55:13 CEST 2022

Copying cl_11_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_11_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688108      cl_11_pbe+        180                                             00:52:58      0:0 
5688108.bat+      batch         36        1   00:00:02          0   00:00:02   00:52:58      0:0 
5688108.ext+     extern        180        6   00:00:00          0   00:00:00   00:52:58      0:0 
5688108.0      mrchem.x        180       30   00:52:25         29   02:27:53   00:52:32      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688108                                                                          
5688108.bat+     19336K          0     19336K        0              0          0 
5688108.ext+          0          0          0        0              0          0 
5688108.0     13310728K         29 5049726429        0             12          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688108                                                                                               
5688108.bat+       16.88M               0         16.88M        0.03M                0          0.03M 
5688108.ext+        0.00M               0          0.00M            0                0              0 
5688108.0           6.75M               0          6.73M      118.03M                0         72.48M 

Job 5688108 completed at Thu May 19 20:48:12 CEST 2022
