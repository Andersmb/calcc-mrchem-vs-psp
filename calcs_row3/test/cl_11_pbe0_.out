                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.060263       -0.806490       -0.233277
    1       O            :       -0.202219        0.617419        2.792174
    2       O            :       -0.020806        1.079646       -2.469390
    3       H            :        1.420488        1.428867        3.081491
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              17
 Total electrons         :                                              34
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998936934e-01
    1    1    a          :                              9.999999959062e-01
    2    1    a          :                              9.999999498979e-01
    3    1    a          :                              1.000000001749e+00
    4    1    a          :                              9.999999997435e-01
    5    1    a          :                              9.999999997288e-01
    6    1    a          :                              1.000000000023e+00
    7    1    a          :                              1.000000134602e+00
    8    1    a          :                              9.999999298140e-01
    9    1    a          :                              9.999999264522e-01
   10    1    a          :                              1.000000116552e+00
   11    1    a          :                              9.999997203745e-01
   12    1    a          :                              1.000000223848e+00
   13    1    a          :                              9.999999142800e-01
   14    1    a          :                              1.000003156034e+00
   15    1    a          :                              1.000000265646e+00
   16    1    a          :                              1.000000029416e+00
   17    1    b          :                              9.999998938265e-01
   18    1    b          :                              9.999999765065e-01
   19    1    b          :                              9.999999355052e-01
   20    1    b          :                              9.999999998024e-01
   21    1    b          :                              9.999999999993e-01
   22    1    b          :                              9.999999997290e-01
   23    1    b          :                              9.999999999991e-01
   24    1    b          :                              1.000000057526e+00
   25    1    b          :                              9.999999297570e-01
   26    1    b          :                              9.999999273226e-01
   27    1    b          :                              1.000000106145e+00
   28    1    b          :                              9.999997004323e-01
   29    1    b          :                              1.000000086370e+00
   30    1    b          :                              1.000000056317e+00
   31    1    b          :                              1.000000336373e+00
   32    1    b          :                              9.999999560891e-01
   33    1    b          :                              9.999999896641e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                606.010878006058
 E-N energy              :            (au)              -1660.351604049126
 Coulomb energy          :            (au)                383.547615339202
 Exchange energy         :            (au)                -11.097843102036
 X-C energy              :            (au)                -34.444326290961
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                108.902159240443
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -716.335280096864
 Nuclear energy          :            (au)                108.902159240443
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.074331208564e+02
                         :      (kcal/mol)             -3.811700351216e+05
                         :        (kJ/mol)             -1.594815427984e+06
                         :            (eV)             -1.652909654375e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.462347299430
    1    1    a          :            (au)                -19.062323301813
    2    1    a          :            (au)                -18.574713439296
    3    1    a          :            (au)                 -9.859870883344
    4    1    a          :            (au)                 -7.610603251801
    5    1    a          :            (au)                 -7.604657236683
    6    1    a          :            (au)                 -7.591774563541
    7    1    a          :            (au)                 -1.168846592434
    8    1    a          :            (au)                 -1.014302765017
    9    1    a          :            (au)                 -0.758416626404
   10    1    a          :            (au)                 -0.590501077331
   11    1    a          :            (au)                 -0.555031231174
   12    1    a          :            (au)                 -0.420199551612
   13    1    a          :            (au)                 -0.352909665589
   14    1    a          :            (au)                 -0.235754951243
   15    1    a          :            (au)                 -0.078908103789
   16    1    a          :            (au)                 -0.014492423107
   17    1    b          :            (au)               -101.462335158366
   18    1    b          :            (au)                -19.062324332344
   19    1    b          :            (au)                -18.574706187136
   20    1    b          :            (au)                 -9.859876706156
   21    1    b          :            (au)                 -7.610604317751
   22    1    b          :            (au)                 -7.604654959964
   23    1    b          :            (au)                 -7.591771804586
   24    1    b          :            (au)                 -1.168843667295
   25    1    b          :            (au)                 -1.014298946528
   26    1    b          :            (au)                 -0.758409541361
   27    1    b          :            (au)                 -0.590474702427
   28    1    b          :            (au)                 -0.555014289068
   29    1    b          :            (au)                 -0.420174821913
   30    1    b          :            (au)                 -0.352900534546
   31    1    b          :            (au)                 -0.235716617755
   32    1    b          :            (au)                 -0.078790051155
   33    1    b          :            (au)                 -0.014492575620
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -353.911042177581
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.830952e+00        -607.433120856421         -6.074331e+02
    1          5.616459e-01        -610.639348942079         -3.206228e+00
    2          3.259936e-01        -610.812478755838         -1.731298e-01
    3          1.965135e-01        -610.844619208369         -3.214045e-02
    4          9.381901e-02        -610.862999106929         -1.837990e-02
    5          6.984202e-02        -610.872965040682         -9.965934e-03
    6          2.968839e-02        -610.875040003243         -2.074963e-03
    7          2.133993e-02        -610.876160201071         -1.120198e-03
    8          1.533207e-02        -610.876491583850         -3.313828e-04
    9          1.456487e-02        -610.876656426802         -1.648430e-04
   10          1.092362e-02        -610.876794848293         -1.384215e-04
   11          1.001538e-02        -610.876891511332         -9.666304e-05
   12          7.803127e-03        -610.876933794318         -4.228299e-05
   13          2.576453e-03        -610.876943505515         -9.711197e-06
   14          1.688044e-03        -610.876949047804         -5.542289e-06
   15          1.694638e-03        -610.876953030405         -3.982601e-06
   16          1.118853e-03        -610.876954445081         -1.414676e-06
   17          1.088814e-03        -610.876955458268         -1.013186e-06
   18          6.217125e-04        -610.876955916573         -4.583054e-07
   19          5.572452e-04        -610.876956259620         -3.430464e-07
   20          7.059803e-04        -610.876956612751         -3.531311e-07
   21          6.368846e-04        -610.876956837142         -2.243911e-07
   22          5.155950e-04        -610.876957012058         -1.749165e-07
   23          3.536463e-04        -610.876957101836         -8.977747e-08
   24          2.877682e-04        -610.876957165880         -6.404366e-08
   25          2.479666e-04        -610.876957215743         -4.986305e-08
   26          1.548405e-04        -610.876957249651         -3.390835e-08
   27          1.247678e-04        -610.876957265759         -1.610840e-08
   28          9.539476e-05        -610.876957275198         -9.438281e-09
---------------------------------------------------------------------------
                      SCF converged in 28 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.060263       -0.806490       -0.233277
    1       O            :       -0.202219        0.617419        2.792174
    2       O            :       -0.020806        1.079646       -2.469390
    3       H            :        1.420488        1.428867        3.081491
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                609.569990147651
 E-N energy              :            (au)              -1668.067892747663
 Coulomb energy          :            (au)                384.078546849242
 Exchange energy         :            (au)                -11.035202362686
 X-C energy              :            (au)                -34.324558402184
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                108.902159240443
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -719.779116515641
 Nuclear energy          :            (au)                108.902159240443
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.108769572752e+02
                         :      (kcal/mol)             -3.833310750841e+05
                         :        (kJ/mol)             -1.603857219193e+06
                         :            (eV)             -1.662280810259e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.869660935906
    1    1    a          :            (au)                -19.289022644551
    2    1    a          :            (au)                -19.246389461279
    3    1    a          :            (au)                 -9.679765049818
    4    1    a          :            (au)                 -7.415780370060
    5    1    a          :            (au)                 -7.410904470715
    6    1    a          :            (au)                 -7.401965844566
    7    1    a          :            (au)                 -1.137356090626
    8    1    a          :            (au)                 -1.061033952053
    9    1    a          :            (au)                 -0.808972756159
   10    1    a          :            (au)                 -0.574252555377
   11    1    a          :            (au)                 -0.507808791472
   12    1    a          :            (au)                 -0.499958781626
   13    1    a          :            (au)                 -0.446302325787
   14    1    a          :            (au)                 -0.381497817017
   15    1    a          :            (au)                 -0.346873443931
   16    1    a          :            (au)                 -0.281831917729
   17    1    b          :            (au)               -101.869660936773
   18    1    b          :            (au)                -19.289022614365
   19    1    b          :            (au)                -19.246389488267
   20    1    b          :            (au)                 -9.679765050122
   21    1    b          :            (au)                 -7.415780372378
   22    1    b          :            (au)                 -7.410904470046
   23    1    b          :            (au)                 -7.401965844849
   24    1    b          :            (au)                 -1.137356086070
   25    1    b          :            (au)                 -1.061033929696
   26    1    b          :            (au)                 -0.808972771785
   27    1    b          :            (au)                 -0.574252549670
   28    1    b          :            (au)                 -0.507808750665
   29    1    b          :            (au)                 -0.499958799002
   30    1    b          :            (au)                 -0.446302323335
   31    1    b          :            (au)                 -0.381497798425
   32    1    b          :            (au)                 -0.346873492756
   33    1    b          :            (au)                 -0.281831932926
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -356.718754419801
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.094023       -1.672713       -1.018099
 Magnitude               :            (au)                        1.960442
                         :         (Debye)                        4.982947
---------------------------------------------------------------------------
 Nuclear vector          :        0.660765        1.295064        1.698050
 Magnitude               :            (au)                        2.235436
                         :         (Debye)                        5.681911
---------------------------------------------------------------------------
 Total vector            :        0.566742       -0.377649        0.679951
 Magnitude               :            (au)                        0.962367
                         :         (Debye)                        2.446092
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 52m 27s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
